spec.html

```diff
@@ -25522,6 +25521,7 @@
       <p>The Date constructor is the %Date% intrinsic object and the initial value of the `Date` property of the global object. When called as a constructor it creates and initializes a new Date object. When `Date` is called as a function rather than as a constructor, it returns a String representing the current time (UTC).</p>
       <p>The `Date` constructor is a single function whose behaviour is overloaded based upon the number and types of its arguments.</p>
       <p>The `Date` constructor is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified `Date` behaviour must include a `super` call to the `Date` constructor to create and initialize the subclass instance with a [[DateValue]] internal slot.</p>
+      <p>The `length` property of the `Date` constructor function is 7.</p>
```

Comment:
>And I'm also not sure whether or not it's correct to place the length declaration under a specific emu-clause, because it should actually apply to the whole constructor function and not just to one of the function overloads.

Exactly, those overload clauses are not describing distinct functions.  They are simply a factoring of the algorithm of the single constructor algorithm that makes it easier to understand that how different argument combinations are handled. 

It is not correct to associate the constructor function's `length` with any one of those overloads.  This sort of over-load confusion is one of the reasons that I preserved the legacy conventions of specifying  the `length` value in the constructor properties subclause.

If you want to move it from there, the correct place to put it would be in [20.3.2](https://tc39.github.io/ecma262/#sec-date-constructor) and to handle all other explicit `length` statements similarly for other overloaded constructors.